internal/provider/provider.go

```diff
@@ -104,6 +152,7 @@ func (p *infisicalProvider) Configure(ctx context.Context, req provider.Configur
 	// Machine Identity
 	clientId := os.Getenv("INFISICAL_UNIVERSAL_AUTH_CLIENT_ID")
 	clientSecret := os.Getenv("INFISICAL_UNIVERSAL_AUTH_CLIENT_SECRET")
+	identityId := os.Getenv("INFISICAL_MACHINE_IDENTITY_ID")
```

Comment:
I think it makes sense to create a file with constants (`constants.go`), if we don't have one already. We can put all these static strings like `INFISICAL_MACHINE_IDENTITY_ID` in there, so we don't mix them up if we need to use them elsewhere